lib/common/node.go

```diff
@@ -134,10 +134,15 @@ func (v *Validator) RemoveValidators(validators ...*Validator) error {
 }
 
 func (v *Validator) MarshalJSON() ([]byte, error) {
+	var neighbors = make(map[string]struct{})
+	for _, neighbor := range v.validators {
+		neighbors[neighbor.Address()] = struct{}{}
+	}
 	return json.Marshal(map[string]interface{}{
-		"address":  v.Address(),
-		"alias":    v.Alias(),
-		"endpoint": v.Endpoint().String(),
+		"address":    v.Address(),
+		"alias":      v.Alias(),
+		"endpoint":   v.Endpoint().String(),
+		"validators": neighbors,
 		//"validators": v.validators,
```

Comment:
👍 Looks good to me!
There are no tests about marshaling, so checking these code is a bit difficult.
I added related tests #134 already, so we could be able to check it after merging this and #134.

ps. comment seems to be removed.